paas_prometheus_exporter/output.tf

```diff
@@ -1,3 +1,9 @@
-output endpoint {
-  value = cloudfoundry_route.paas_prometheus_exporter.endpoint
+output exporter {
+  value = {
+     endpoint = cloudfoundry_route.paas_prometheus_exporter.endpoint
+     name     = cloudfoundry_app.paas_prometheus_exporter.name
+     scheme   = "https"
+     honor_labels = false
```

Comment:
do we need to set honor_labels per exporter? can't we leave it as false by default in prometheus?

Reply:
yes I will look at that